sammba/registration/struct.py

```diff
@@ -148,7 +148,9 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
 
     brain_extraction_in_files = []
     for n, anat_file in enumerate(copied_anat_filenames):
-        out_unifize = unifize(in_file=anat_file, outputtype='NIFTI_GZ',
+        out_unifize = unifize(in_file=anat_file,
+                              out_file='%s_Unifized_for_brain_mask.nii.gz',
```

Comment:
you can even remove the .nii.gz, since only the basename without extension is taken into account from what I tried with nipype